stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -12,22 +12,15 @@
 
 /// A fixed-width integer that is twice the size of its base type.
 @_fixed_layout // FIXME(sil-serialize-all)
-public struct DoubleWidth<Base : FixedWidthInteger> :
-  FixedWidthInteger, _ExpressibleByBuiltinIntegerLiteral
-  where Base.Words : Collection, Base.Magnitude.Words : Collection {
+public struct DoubleWidth<Base : FixedWidthInteger> : _ExpressibleByBuiltinIntegerLiteral
```

Comment:
So... Do we officially drop the 80 column limit?

Reply:
Oh, check this out! I have an argument for keeping the 80-columns right here ;-)
![_stdlib__squash_some_warnings_in_stdlib_by_airspeedswift_ _pull_request__12753_ _apple_swift](https://user-images.githubusercontent.com/64011/33451123-d14aca9c-d5c2-11e7-81c3-adf588c634dd.png)

Reply:
Nah I'm just flirting with flaunting it occasionally...